docs/flow/monitoring/debugging.md

```diff
@@ -6,3 +6,53 @@ weight: 300
 ---
 
 # Debugging
+
+There are two primary ways to debug issues with Grafana Agent Flow:
+
+1. Through the Grafana Agent Flow UI
+2. Through examining logs
```

Comment:
Do we want to also add a third bullet-point, where users might want to check the component-level metrics?

Reply:
Good idea, I'll write a section for that 👍

Reply:
Actually, thinking about it, I don't know if component-level metrics helps _debug_ issues (rather, they're more useful for _identifying_ issues). WDYT?

Reply:
For now, I'm going to leave out checking metrics since I'm not positive that's ever helpful for resolving/debugging issues